memory-viz/src/tests/plugins/pretty_svg.ts

```diff
@@ -11,7 +11,10 @@ export const serialize: NewPlugin["serialize"] = (val: HTMLElement) => {
 
 export const test: NewPlugin["test"] = (val: any) => {
     const root = parse(val);
-    return root.firstChild.rawTagName === "svg";
+    if (root.firstChild !== undefined) {
```

Comment:
It's a bit cleaner here to just use the `?.` operator to acccess `rawTagName`, since `undefined === "svg"` will evaluate to `false`.